lua/apisix/plugins/limit-count.lua

```diff
@@ -82,6 +88,8 @@ function _M.check_schema(conf)
 
         conf.redis_port = conf.redis_port or 6379
         conf.redis_timeout = conf.redis_timeout or 1000
+        conf.redis_pool_size = conf.redis_pool_size or 1000
+        conf.redis_backlog = conf.redis_backlog or 1000
```

Comment:
Please set the backlog to `nil` to disable the feature by default.
Two reasons:
1. the backlog feature in cosocket is considered experimental.
2. it isn't a smart idea to queue limitation count. This will be make the result incorrect. The operation should be finished ASAP to make the limitation up to date.

Reply:
Thank you for reminding @spacewander

Reply:
> Please set the backlog to `nil` to disable the feature by default.
> Two reasons:
> 
> 1. the backlog feature in cosocket is considered experimental.
> 2. it isn't a smart idea to queue limitation count. This will be make the result incorrect. The operation should be finished ASAP to make the limitation up to date.

thanks，set backlog `nil` by default , done